packages/js/product-editor/src/components/feedback-bar/test/feedback-bar.test.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { createElement } from '@wordpress/element';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { FeedbackBar } from '../feedback-bar';
+import { useFeedbackBar } from '../../../hooks/use-feedback-bar';
+
+jest.mock( '../../../hooks/use-feedback-bar', () => ( {
+	...jest.requireActual( '../../../hooks/use-feedback-bar' ),
+	useFeedbackBar: jest.fn(),
+} ) );
+
+jest.mock( '@woocommerce/tracks', () => ( {
+	...jest.requireActual( '@woocommerce/tracks' ),
+	recordEvent: jest.fn(),
+} ) );
+
+describe( 'FeedbackBar', () => {
+	beforeEach( () => {
+		jest.clearAllMocks();
+	} );
+	it( 'should render empty name rows when names are empty', () => {
```

Comment:
The test case name mentions rendering empty name rows, but the test is about turning off the feedback bar. Is this expected?

Reply:
Good catch! Copy pasta error. Fixed in [aef7ae2](https://github.com/woocommerce/woocommerce/pull/48245/commits/aef7ae2feda1bba9739e3589dc6dd0320937f6ba)[aef7ae2](https://github.com/woocommerce/woocommerce/pull/48245/commits/aef7ae2feda1bba9739e3589dc6dd0320937f6ba)